pom.xml

```diff
@@ -43,8 +43,8 @@
 
         <gcp-opentelemetry.version>0.33.0</gcp-opentelemetry.version>
 
-        <azure-monitor-opentelemetry-autoconfigure.version>1.0.0-beta.2</azure-monitor-opentelemetry-autoconfigure.version>
-        <quarkus-azure-services-bom.version>1.0.2</quarkus-azure-services-bom.version>
+        <azure-monitor-opentelemetry-autoconfigure.version>1.1.0</azure-monitor-opentelemetry-autoconfigure.version>
+        <quarkus-azure-services-bom.version>1.1.1</quarkus-azure-services-bom.version>
```

Comment:
This Maven property does not seem to be used.

I haved removed it in #334 and the CI is happy. cc @brunobat 

@ninckblokje If there is a dependency conflict between the Quarkus Opentelemetry Exporter for Microsoft Azure and Camel dependencies, you could perhaps produce dependency trees with Camel dependencies and with/without the Quarkus Opentelemetry Exporter for Microsoft Azure to help investigate.

Reply:
I have rebased my branch and removed the property. The conflict was caused by using an old version quarkus-azure-services, which pulled in old Azure dependencies.